Server/integrations/twilio_sms/twilio2FA.py

```diff
@@ -104,10 +104,16 @@ def authenticate(self, configurationAttributes, requestParameters, step):
                 return False
 
             try:
-                mobile_number = foundUser.getAttribute("phoneNumberVerified")
-                if isinstance(mobile_number, JSONArray):
-                    mobile_number=mobile_number.get(0)
-                    print "TwilioSMS, Message will be sent to number ", mobile_number
+                isVerified = foundUser.getAttribute("phoneNumberVerified")
+                if isVerified:
+                    mobile_number = foundUser.getAttribute("employeeNumber")
+                if  mobile_number == None:
+                    mobile_number = foundUser.getAttribute("mobile")
+                if  mobile_number == None:
+                    mobile_number = foundUser.getAttribute("telephoneNumber")
+                if  mobile_number == None: 
+                    print "TwilioSMS, Error finding mobile number for' % (user_name)
+                    
```

Comment:
After getting mobile_number you should restore lines:
```
                if isinstance(mobile_number, JSONArray):
                    mobile_number=mobile_number.get(0)
print "TwilioSMS, Message will be sent to number ", mobile_number
```

Reply:
i think this block is no more useful because the mobile_number is no more save in ldap as a list.

Reply:
Hence we don't need to test if it is an instance of a jsonArray.

Reply:
make sense?